src/client/panes/plugin_pane.rs

```diff
@@ -13,6 +14,7 @@ pub struct PluginPane {
     pub position_and_size_override: Option<PositionAndSize>,
     pub send_plugin_instructions: SenderWithContext<PluginInstruction>,
     pub max_height: Option<usize>,
+    pub active_at: u64,
```

Comment:
As a small code tweak though, I'd prefer if we used [`Instant`](https://doc.rust-lang.org/std/time/struct.Instant.html)s here. Those are actually guaranteed to never do anything odd around leap-seconds and are much more idiomatic Rust (just in my opinion)

Reply:
I'm on it.